.github/workflows/backend.yml

```diff
@@ -95,51 +40,8 @@ jobs:
       - run: clojure -X:dev:ee:ee-dev:drivers:drivers-dev:test:eastwood
         name: Run Eastwood linter
 
-  # Because it's not possible to conditionally run only `java-11-ee` test in the draft mode,
-  # we have to extract that job manually here. Backend developers have requested that this
-  # test runs at all times to give them an early warning sign is something is broken.
-  be-tests-pre-check:
-    needs: [files-changed, static-viz-files-changed]
-    if: |
-      !cancelled() &&
-      github.event.pull_request.draft == true &&
-      (needs.files-changed.outputs.backend_all == 'true' || needs.static-viz-files-changed.outputs.static_viz == 'true')
-    runs-on: ubuntu-22.04
-    name: Backend Tests Pre-Check
-    timeout-minutes: 60
-    steps:
-      - uses: actions/checkout@v4
-      - name: Prepare front-end environment
-        uses: ./.github/actions/prepare-frontend
-      - name: Prepare back-end environment
-        uses: ./.github/actions/prepare-backend
-        with:
-          m2-cache-key: backend-pre-check
-
-      - run: yarn install --frozen-lockfile --prefer-offline
-      - name: Build static viz frontend
-        run: yarn build-static-viz
-
-      - name: Run tests
-        run: clojure -X:dev:ci:test:ee:ee-dev
-
-      - name: Publish Test Report (JUnit)
-        uses: dorny/test-reporter@v1
-        if: failure()
-        with:
-          path: "target/junit/**/*_test.xml"
-          name: JUnit Test Report (Backend Tests Pre-Check)
-          reporter: java-junit
-          list-suites: failed
-          list-tests: failed
-          fail-on-error: false
```

Comment:
@dpsutton , can you confirm that this job is redundant if we're moving all tests to run on draft PRs? (i.e. are all these tests included in the `be-tests` job below?